vyper/venom/passes/make_ssa.py

```diff
@@ -0,0 +1,170 @@
+from vyper.utils import OrderedSet
+from vyper.venom.analysis import calculate_cfg, calculate_liveness
+from vyper.venom.basicblock import IRBasicBlock, IRInstruction, IROperand, IRVariable
+from vyper.venom.dominators import DominatorTree
+from vyper.venom.function import IRFunction
+from vyper.venom.passes.base_pass import IRPass
+
+
+class MakeSSA(IRPass):
+    """
+    This pass converts the function into Static Single Assignment (SSA) form.
+    """
+
+    dom: DominatorTree
+    defs: dict[IRVariable, OrderedSet[IRBasicBlock]]
+
+    def _run_pass(self, ctx: IRFunction, entry: IRBasicBlock) -> int:
+        global count
```

Comment:
why global? i think we can use `self.count` instead?

Reply:
it was a debugging leftover. Not used.